src/System.IO.FileSystem/tests/Base/AllGetSetAttributes.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    // Tests that are valid for File, FileInfo, and DirectoryInfo
+    public abstract class AllGetSetAttributes : BaseGetSetAttributes
+    {
+        [Fact]
+        public void NullParameters()
+        {
+            Assert.Throws<ArgumentNullException>(() => Get(null));
+            Assert.Throws<ArgumentNullException>(() => Set(null, FileAttributes.Normal));
```

Comment:
Any parameter names we can verify for these and other argument exceptions in these tests?